.changeset/slimy-bears-stare.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/react-brand': minor
```

Comment:
I think this could be a patch instead of minor. Usually minor is used for new components or major features.

Reply:
I see, good call! I swapped this out for patch.